api/src/main/java/io/strimzi/api/kafka/model/CertSecretSource.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents a certificate inside a Secret
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = true,
+        builderPackage = "io.strimzi.api.kafka.model"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CertSecretSource implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    protected String name;
+    protected String cert;
+
+    protected Map<String, Object> additionalProperties;
+
+    @Description("The name of the Secret containing the certificate")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Description("The name of the file certificate in the Secret")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getCert() {
```

Comment:
I think in general k8s doesn't use abbreviations like "cert" in its schemas (e.g. https://v1-10.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#certificatesigningrequestcondition-v1beta1-certificates), so neither should we.

Reply:
Yeah, I was in doubt about that.